src/ucp/wireup/select.c

```diff
@@ -827,19 +831,20 @@ static uint64_t ucp_ep_get_context_features(const ucp_ep_h ep)
     return ep->worker->context->config.features;
 }
 
-static double ucp_wireup_rma_score_func(const ucp_worker_iface_t *wiface,
+static double ucp_wireup_rma_score_func(const ucp_wireup_criteria_t *self,
```

Comment:
maybe pass 'void*' user-defined argument instead of self (also, as last param) ?

Reply:
btw, shouldn't it be a part of criteria? Otherwise we will need to define new structure controlling lanes selection and only bw-related functions will pass it to the general selection routine  `ucp_wireup_select_transport`  (others will have to pass `NULL`)

Reply:
anyway the BW stuff should not be part of the general criteria so need to define a new struct